src/query/plan/vertex_count_cache.hpp

```diff
@@ -76,6 +76,40 @@ class VertexCountCache {
     return bounds_vertex_count.at(bounds);
   }
 
+  int64_t EdgesCount(storage::EdgeTypeId edge_type) {
+    if (edge_type_edge_count_.find(edge_type) == edge_type_edge_count_.end())
+      edge_type_edge_count_[edge_type] = db_->EdgesCount(edge_type);
+    return edge_type_edge_count_.at(edge_type);
+  }
+
+  int64_t EdgesCount(storage::EdgeTypeId edge_type, storage::PropertyId property) {
+    auto key = std::make_pair(edge_type, property);
+    if (edge_type_property_edge_count_.find(key) == edge_type_property_edge_count_.end())
+      edge_type_property_edge_count_[key] = db_->EdgesCount(edge_type, property);
+    return edge_type_property_edge_count_.at(key);
+  }
+
+  int64_t EdgesCount(storage::EdgeTypeId edge_type, storage::PropertyId property, const storage::PropertyValue &value) {
+    auto edge_type_prop = std::make_pair(edge_type, property);
+    auto &value_edge_count = property_value_edge_count_[edge_type_prop];
+    // TODO: Why do we even need TypedValue in this whole file?
```

Comment:
Can you explain / resolve the issue here?

Reply:
I think I just copied this from "vertices". As this is a draft still, I haven't checked the whole code. I'm going to see who even commented this under the same `Vertices()` method